protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -101,6 +113,55 @@ async fn test_movement_client_should_successfully_call_lock() -> Result<(), anyh
 	{ let movement_client = harness.movement_client_mut().expect("Failed to get MovementClient");
 
 	let _ = movement_client.publish_for_test();
+
```

Comment:
ditto

Reply:
Removed all stderr and standout logging from `publish_for_test`.